docs/releases.md

```diff
@@ -0,0 +1,53 @@
+# Releases
+
+This document describes how the release flow looks like and how to create a marketplace release.
+
+## Release process
+
+The release process is based on Git tags. Once tag is created the release process will start automatically.
+
+### Versioning
+
+Marketplace versioning should follow the Kyma versions. So Marketplace in version 1.1.1 will install Kyma in version 1.1.1. Versions with suffix will generate pre-release.
```

Comment:
Marketplace versioning should follow Kyma versions. Marketplace in version 1.1.1 installs Kyma in version 1.1.1. Versions with a suffix generate pre-releases.